docs/rfc/2022-07-05-mo-cluster.md

```diff
@@ -87,6 +87,44 @@ It is worth noting that `MatrixOneCluster` object is not necessary to create a m
 - advanced user can ignore `MatrixOneCluster` object perform fine-grained control over `*Set` directly;
 - it is trivial to support heterogenous cluster: one simply deploy several `*Set` objects with different Pod specification;
 
+An overview of the deployment topology:
+
+```mermaid
+flowchart TD
+  subgraph k8s
+    subgraph Cluster
+    direction TB
+    LoadBalancer
+    subgraph CN
+      direction LR
+      CN-0 --o CN-1
+      CN-1 --> |scale-out| CN-2
+      style CN-2 fill:#bbf,stroke:#f66,stroke-width:2px,color:#fff,stroke-dasharray: 5 5
+    end
+    subgraph DN
+      direction RL
+      DN-1 --> |standby| DN-0
+      style DN-1 fill:#bbf,stroke:#f66,stroke-width:2px,color:#fff,stroke-dasharray: 5 5
+    end
+    subgraph LogService
+      direction LR
+      LogStore-0 <--> LogStore-1
+      LogStore-1 <--> LogStore-2
+      LogStore-2 <--> LogStore-0
+    end
+    end
+      Grafana --> |Query Metrics| Cluster
```

Comment:
I prefer use `WebUI` instead of `Grafana`

Reply:
Looks good, thanks!